src/library.js

```diff
@@ -3769,7 +3769,7 @@ function wrapSyscallFunction(x, library, isWasi) {
   if (!WASMFS) {
     library[x + '__deps'].push('$SYSCALLS');
   }
-#if PTHREADS
+#if SHARED_MEMORY
```

Comment:
I think this `#if` could be dropped altogether? The proxying directives can exist independent of the build configuration one is running with, and don't need to be gated to only be specified in multithreaded builds?

Reply:
Good idea. Done.